react/components/Page.jsx

```diff
@@ -1,22 +1,23 @@
-var React = require('react');
-var Link = require('react-router').Link;
-var Sidebar = require('./Sidebar.jsx');
+var React = require(`react`);
+var Link = require(`react-router`).Link;
+var Sidebar = require(`./Sidebar.jsx`);
 
 var elements = (function() {
-  var randomstring = require('randomstring');
-  var elements = [];
+  var randomstring = require(`randomstring`);
+  var elems = [];
   var i=100;
+
   while(i--) {
-    elements.push(randomstring.generate());
+    elems.push(randomstring.generate());
   }
-  return elements;
-}());
+  return elems;
```

Comment:
this should not be a conflict, ever, so that's more a motivation to turn that rule off on the mofo style repo